package.json

```
@@ -7,6 +7,7 @@
     "node": ">=16 <17"
   },
   "dependencies": {
+    "@djpfs/react-vlibras": "^2.0.2",
```

Esqueci de comentar, mas neste projeto usamos a versão pinada das dependências. Então será preciso mudar aqui e refazer o `npm install`

```suggestion
    "@djpfs/react-vlibras": "2.0.2",
```